src/TestHelper/SqlHelpers.cs

```diff
@@ -12,13 +12,9 @@ public static void ExecuteCommand(this DbConnection connection, string script, s
             {
                 command.CommandText = script;
                 command.AddParameter("tablePrefix", $"{tablePrefix}_");
-                if (schema == null)
+                if (command is System.Data.SqlClient.SqlCommand)
```

Comment:
In what way? Currently isn't SQL Server the only engine that supports/needs a schema? MySQL doesn't but it can tolerate extra unused parameters. Oracle throws if you give it parameters you're not using.

Would you prefer passing in the sqlVariant to these? Or making SqlHelpers a non-static class that takes a sqlVariant in its constructor?